src/requirement-types.d.ts

```diff
@@ -7,6 +7,11 @@ type RequirementCommon = {
   readonly source: string;
   /** If this is set to true, then an edge to the course doesn't count towards double counting. */
   readonly allowCourseDoubleCounting?: true;
```

Comment:
unrelated to your PR, but I just noticed this. why is the type of allowCourseDoubleCounting true?

Reply:
It's an optional field. (See the ?), so when it doesn't exist it just means false

Reply:
ah got it, ty!!